src/main/java/com/cgdp/library/service/AuthorService.java

```diff
@@ -0,0 +1,24 @@
+package com.cgdp.library.service;
+
+import com.cgdp.library.dto.author.CreateAuthorResponseDTO;
+import com.cgdp.library.entity.AuthorEntity;
+import com.cgdp.library.repository.AuthorRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@AllArgsConstructor
+public class AuthorService {
+
+    private AuthorRepository authorRepository;
+
+    @Transactional
+    public CreateAuthorResponseDTO createAuthor(String authorName) {
```

Comment:
I meant that here we should return an `AuthorDTO` so you need to create the object, you can use ModelMapper to map it from entity to DTO
```suggestion
    public AuthorDTO createAuthor(String authorName) {
```

Imagine that this method needs to be used in any other context, returning a response DTO (API layer) here doesn't make sense